src/state/slices/swapSlice/selectors.ts

```diff
@@ -0,0 +1,26 @@
+import type { Swap } from '@shapeshiftoss/swapper'
+import { createSelector } from 'reselect'
+
+import { swapSlice } from './swapSlice'
+
+import { selectQuoteIdParamFromFilter } from '@/state/selectors'
+
+export const selectSwaps = swapSlice.selectors.selectSwaps
+
+export const selectSwapById = createSelector(selectSwaps, swaps => {
```

Comment:
That looks wrong:

- This should be `selectSwapsById` 
- If we need a selector to mapreduce over swaps, we probably don't need such selector and could just store a `byId` (and `ids`) alongside array collection, wdyt?

Reply:
Nice catch, stored byId in [fbcd366](https://github.com/shapeshift/web/pull/9625/commits/fbcd366ded019ee7c8586022801e2a9625cc80dd)